mini_project_1.ipynb

```diff
@@ -0,0 +1,175 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'the'"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from pattern.web import *\n",
+    "import string\n",
+    "\n",
+    "christianity=URL(\"http://www.gutenberg.org/cache/epub/8294/pg8294.txt\").download()\n",
```

Comment:
You can move the declaration of the variable to the line before you process them, such as:
```
christianity=URL("http://www.gutenberg.org/cache/epub/8294/pg8294.txt").download()
process_file(christianity)
```
This improves the readability of your code by grouping related things.